_posts/2019-12-20-this-blog-post-contains-important-information-for-some-percentage-of-its-readers.markdown

```diff
@@ -0,0 +1,150 @@
+that were meaningful to you and someone else could understand and offer a meaningful response, and this made you
+feel uncomfortable at first but you really loved it. Your experience of college was maybe a bit different than you
+expected - certainly nothing like the movies you watched as a kid. You got really into a weird hobby, maybe, and
+didn't end up going to too many parties.
+
+Relationships and intimacy have always been a bit fraught for you. You frequently find yourself trying to figure
+out what you're supposed to be feeling, and you work hard to show your partner the person you want to be. Maybe
+when you first moved in with someone you had a bit of a freakout that has made you nervous to think about ever
+since, or maybe meeting a significant other's family has been so anxiety-inducing as to prove impossible. Maybe you
+have a string of failed relationships behind you, each of them starting out so promisingly and then imploding
+catastrophically and confusingly once the initial honeymoon phase is over and it's time to really get to know and
+grow with another person. You don't know how to do that, and every time you try you seem to make someone upset.
+
+You're a software engineer at least partly because computers are much easier to communicate with than people. When
+a computer doesn't understand you it's because you've done something wrong. When a computer says something
+incorrect it's also because you've done something wrong. You can step through and rigorously debug the problem,
+identifying and isolating the mistake. The more you played around with tech the more you liked it - and when you
+discovered programming it blew your mind, because you _intuitively_ understood the relational nature of computation
+and found yourself suddenly gifted with superpowers.
+
+When you entered the working world you were REALLY confused, though. You were able to complete a week's worth of
+assignments in an afternoon. It was really obvious to you who was working intelligently and who was not, and you
+had to learn, perhaps painfully, that sharing that kind of feedback isn't always constructive. Maybe one day you
+got some criticism about something you did and it really hurt in ways you didn't expect -- maybe you found yourself
+reacting defensively, and the next thing you know you're in a meeting with HR over what to you was clearly all a
+big misunderstanding, but nobody is smiling. Or maybe instead someone recognized that you were having a tough time,
+and stepped in to defuse the situation.
+
+Regardless, your agency was compromised. You found yourself resenting the arbitrary and unwritten rules of the
+professional world: you have to be at your desk and looking busy even when you're just thinking; let people say
+things that are not true if correcting them publicly will embarrass someone; people will ignore your advice and
+then something will break because of it and it's very important not to yell at them about it.
```

Comment:
"Yell at them" might be too strong, I'm not sure. Up to you:

```suggestion
then something will break because of it and it's very important not to get too upset.
```